lib/common/transforms.js

```diff
@@ -40,6 +40,10 @@ function isAsset(prop) {
 	return prop.attributes.category === 'asset';
 }
 
+function isContent(prop) {
```

Comment:
This should probably be `isAsset` or the conditional below should be checking for 'content'?